packages/react/src/primitives/shared/constants.ts

```diff
@@ -41,6 +41,9 @@ export enum ComponentClassNames {
   Label = 'amplify-label',
   Link = 'amplify-link',
   Loader = 'amplify-loader',
+  Menu = 'amplify-menu',
+  MenuItem = 'amplify-menu__item',
+  MenuTrigger = 'amplify-menu-trigger',
```

Comment:
What do you think of adding a comment here? At least for me, this is where I feel confused and had the question why `'amplify-menu-trigger'`  instead of `'amplify-menu__trigger'`. For external users, I think we have an awesome `Target Classes` section for them to reference

Reply:
After online discussion, I've updated the classnames to `amplify-menu-content` and `amplify-menu-trigger` to make it more readable.